scripts/deploy-ebpf.ps1

```diff
@@ -26,52 +26,125 @@ $build_directory=".\x64\Debug"
 [System.Collections.ArrayList]$test_files=@(
     "api_test.exe",
     "api_test.pdb",
-    "bindmonitor_um.dll",
-    "bindmonitor_um.pdb",
-    "bpf2c_test_wrapper.dll",
-    "bpf2c_test_wrapper.pdb",
-    "bpftool.pdb",
-    "bpftool_tests.exe",
-    "bpftool_tests.pdb",
     "bindmonitor.o",
     "bindmonitor_km.sys",
     "bindmonitor_ringbuf.o",
+    "bindmonitor_ringbuf_um.dll",
+    "bindmonitor_ringbuf_um.dll_um.pdb",
+    "bindmonitor_ringbuf_um.dll_um.pdb.sys",
```

Comment:
why does the filename have ".pdb.sys"?  This is confusing.

Reply:
Bug in the script I used to generate this file.